Multisig/App/AppDelegate+Messaging.swift

```diff
@@ -37,6 +36,10 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
             App.shared.intercomConfig.pushNotificationUserInfo = userInfo
         }
 
+        if SafeDeploymentController.isSafeCreatedNotification(userInfo){
+            SafeDeploymentController.handleSafeCreatedNotification(userInfo: userInfo)
+        }
```

Comment:
this `if` does not prevent the `remote notification handler` from handling the notification. Do we want to make it exclusive?